src/test/java/org/minions/devfund/lourdes/MovieDatabaseTest.java

```diff
@@ -0,0 +1,161 @@
+package org.minions.devfund.lourdes;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Implemented tests for {@link MovieDatabase}.
+ */
+public class MovieDatabaseTest {
+
+    private MovieDatabase movieDatabase;
+    private String movieName1;
+    private String[] actors1;
+
+    /**
+     * Method that setup all initial values.
+     */
+    @Before
+    public void setup() {
+        movieDatabase = new MovieDatabase();
+        movieName1 = "Sexto sentido";
+        actors1 = new String[]{"Bruce Willis", "Haley Joel"};
+    }
+
+    /**
+     * Verify add new movie to the list.
+     */
+    @Test
+    public void testAddMovie() {
+        MovieDatabase movieDatabase = new MovieDatabase();
+        movieDatabase.addMovie(movieName1, actors1);
+        assertEquals(movieDatabase.getMovieList().get(0).getName(), movieName1);
+    }
+
+    /**
+     * Verify add a duplicated movie.
+     */
+    @Test
+    public void testAddExistMovie() {
+        final int totalMovie = 1;
+        movieDatabase.addMovie(movieName1, actors1);
+        movieDatabase.addMovie(movieName1, actors1);
+        assertEquals(movieDatabase.getMovieList().size(), totalMovie);
+    }
+
+    /**
+     * Verify add a new movie with new actors.
+     */
+    @Test
+    public void testAddMovieNewActors() {
+        movieDatabase.addMovie(movieName1, actors1);
+        assertEquals(actors1.length, movieDatabase.getActorList().size());
+        for (String actor : actors1) {
+            assertEquals(movieDatabase.getActorByName(actor).getName(), actor);
+        }
+    }
+
+    /**
+     * Verify add a second movie.
+     */
+    @Test
+    public void testAddMovie2() {
+        final String movieName2 = "Duro de Matar";
+        final String[] actors2 = new String[]{"Bruce Willis", "Alam Rickman"};
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Alam Rickman" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)